### ISSUE TITLE
Can't perform init

### ISSUE THREAD

**Issue by utgarda** [solutious/rudy#55]
Trying to switch to a new account, can't get past "init" step.

Checked S3 access through web-UI, generated and tried many access keys, every time putting it to ~/.rudy/config, but no luck, every time the same response:

> rudy -vvvv --debug init
> Rudy::AWS::SDB::ConnectionError: 401              Unauthorized
> /usr/lib64/ruby/gems/1.8/gems/rudy-0.9.8.017/bin/../lib/rudy/aws/sdb.rb:237:in `call'
> /usr/lib64/ruby/gems/1.8/gems/rudy-0.9.8.017/bin/../lib/rudy/aws/sdb.rb:42:in`list_domains'
> /usr/lib64/ruby/gems/1.8/gems/rudy-0.9.8.017/bin/../lib/rudy/huxtable.rb:56:in `domain_exists?'
> /usr/lib64/ruby/gems/1.8/gems/rudy-0.9.8.017/bin/rudy:244
> /usr/lib64/ruby/gems/1.8/gems/drydock-0.6.9/lib/drydock.rb:180:in`call'
> /usr/lib64/ruby/gems/1.8/gems/drydock-0.6.9/lib/drydock.rb:180:in `call'
> /usr/lib64/ruby/gems/1.8/gems/drydock-0.6.9/lib/drydock.rb:753:in`run!'
> /usr/lib64/ruby/gems/1.8/gems/rudy-0.9.8.017/bin/rudy:313
> /usr/bin/rudy:19:in `load'
> /usr/bin/rudy:19

**Comment by delano** [comment]
Did you create a SimpleDB account?

**Comment by utgarda** [comment]
Yep, as far as I understand. I can use amazon web UI or S3Fox to create buckets. That's the requirement, right?

**Comment by utgarda** [comment]
Sorry, this is the second time I'm coming with false alarms. I didn't subscribe to SimpleDB , only to S3. I'm sorta in a rush, seems like I can't think properly nowdays. So, that could go to a wishlist : maybe a nice diagnostic message would help. And yep, maybe I can do it myself, If I understand what is what in SDB Connector. Thanks for the hint.

**Comment by delano** [comment]
No problem!

**Comment by utgarda** [comment]
Yesterday I made sure my new account has both S3 and SimpleDB services on. I saw several times the message in web UI "You already have access to this web service ". Still today, when I try "rudy init" with the new account's config in my ~/.rudy/config , I get the result described above. Re-generated accesskey and secretkey several times, using accountnum with "-" removed, in form like "452990507090". 

When I use the old config, "rudy init" says "SimpleDB domain rudy_state already exists".
